src/codeu/chat/client/ClientUser.java

```diff
@@ -43,20 +44,25 @@ public ClientUser(Controller controller, View view) {
     this.controller = controller;
     this.view = view;
   }
-
-  // Validate the username string
-  static public boolean isValidName(String userName) {
-    boolean clean = true;
-    if (userName.length() == 0) {
+    
+// Validate the username string
+public boolean isValidName(String userName) {
+	boolean clean = true; 
```

Comment:
small nit about naming. I can't tell from this name what this variable represents. Maybe something like isUniqueUser or isNewUser?